pkgs/by-name/n6/n64recomp/package.nix

```
@@ -0,0 +1,55 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  installShellFiles,
+  unstableGitUpdater,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "n64recomp";
+  version = "0-unstable-2025-02-27";
+
+  src = fetchFromGitHub {
+    owner = "N64Recomp";
+    repo = "N64Recomp";
+    rev = "989a86b36912403cd323de884bf834f2605ea770";
+    hash = "sha256-DlzqixK8qnKrwN5zAqaae2MoXLqIIIzIkReVSk2dDFg=";
+    fetchSubmodules = true;
+  };
+
+  strictDeps = true;
+
+  nativeBuildInputs = [
+    cmake
+    installShellFiles
+  ];
+
+  installPhase = ''
+    runHook preInstall
+
+    installBin {N64Recomp,RSPRecomp}
```

Can we expose `RecompModTool` in this derivation? That way if someone wants to build a mod using nix they can use the upstream nix package.
```suggestion
    installBin {N64Recomp,RSPRecomp,RecompModTool}
```